src/layers/controllers/layersController.ts

```diff
@@ -24,4 +25,13 @@ export class LayersController {
       next(error);
     }
   };
+
+  public addLayerToBest: CreateBestHandler = (req, res, next) => {
```

Comment:
we said we will remove the term "best" from our code, no?

Reply:
fixed in ee20559